packages/suite/src/support/suite/Main.tsx

```diff
@@ -48,6 +51,7 @@ export const Main = ({
                                 <FormatterProvider config={formattersConfig}>
                                     {children}
                                 </FormatterProvider>
+                                {isDesktopApp && <FindBar />}
```

Comment:
huh, it is done this ugly in the whole app? :D lol

I remember we had a proper stub in `browser` field back in avcd. So that this code simply isn't in the builds.